nin/dasBoot/CameraNode.js

```diff
@@ -0,0 +1,31 @@
+(function(NIN) {
+  'use strict';
+
+  class CameraNode extends NIN.Node {
+    constructor(id, options) {
+      super(id, {
+        outputs: {
+          camera: new NIN.CameraOutput()
+        }
+      });
+
+      this.options = options;
+
+      this.cameraController = new CameraController();
+      this.outputs.camera.setValue(this.cameraController.camera);
+      Loader.loadAjax(options.path, response => this.initializeCamera(response));
```

Comment:
How does this loader work together with compilation?
As there won't be a file to load from then?